### PR TITLE
cmd/snap: add 'snap experimental remove-skill'

### PR DESCRIPTION
**Pull request by zyga**
Signed-off-by: Zygmunt Krynicki <zygmunt.krynicki@canonical.com>

### CODE REVIEW

File: cmd/snap/cmd_remove_skill.go [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+		Name string `positional-arg-name:"name" description:"name of the skill slot within the snap"`
+	} `positional-args:"true" required:"true"`
+}
+
+var shortRemoveSkillHelp = i18n.G("Removes a skill from the system")
+var longRemoveSkillHelp = i18n.G(`
+The remove-skill command removes a skill from the system.
+
+This command is only for experimentation with the skill system.
+It will be removed in one of the future releases.
```
Comment (chipaca) [inline cmd/snap/cmd_remove_skill.go]: "in one of the future releases" doesn't read right; maybe "in a future release"?

On the other hand, maybe you want "before 16.04 is released"

Comment (chipaca) [inline cmd/snap/cmd_remove_skill.go]: On the *other* other hand, maybe this should be added by `addExperimentalCommand` itself?

**Comment by chipaca** [comment]
Non-blocking nits. :+1:

File: cmd/snap/cmd_remove_skill.go (hunk 1 above)
Comment (zyga) [inline cmd/snap/cmd_remove_skill.go]: Hmm, let's pick one and stick with it. I need all those command branches landed.

**Comment by pedronis** [comment]
+1

**Pull request merged**